addons/account/static/src/components/bills_upload/bills_upload.xml

```diff
@@ -20,7 +20,10 @@
                 onUploaded.bind="onFileUploaded"
                 onUploadComplete.bind="onUploadComplete">
                 <t t-set-slot="toggler">
-                    <t t-slot="default"/>
+                    <t t-if="props.template">
+                        <t t-call="{{ props.template }}"/>
+                    </t>
+                    <t t-else="" t-slot="default"/>
```

Comment:
To not change your behavior, you can do the <t t-call="{{ props.template }}"/> in your widget

Reply:
in account.AccountFileUploade

Reply:
? this is in the account.AccountFileUploader